package.json

```diff
@@ -27,6 +27,7 @@
     ],
     "dependencies": {
         "ajv": "6.12.3",
+        "async-mutex": "0.2.4",
```

Comment:
You'll need to add this to `packages/node_modules/@node-red/runtime/package.json` as well

Reply:
Got it thanks

Reply:
@knolleary change request made.